servers/interfaces.go

```diff
@@ -772,14 +772,14 @@ func GetUnspends(param Params) map[string]interface{} {
 		return ResponsePack(InvalidParams, "")
 	}
 	type UTXOUnspentInfo struct {
-		Txid  string
-		Index uint32
-		Value string
+		Txid  string `json:"Txid"`
```

Comment:
Txid  string `json:"Txid"` can be 
TxID  string `json:"Txid"`

Reply:
Completed this modification in the third commit.